lib/puppet/module.rb

```diff
@@ -58,7 +58,7 @@ def self.is_module_namespaced_name?(name)
   attr_accessor :dependencies, :forge_name
   attr_accessor :source, :author, :version, :license, :summary, :description, :project_page
 
-  def initialize(name, path, environment)
+  def initialize(name, path, environment, include_prerelease = true)
```

Comment:
Ah, I was asking because everywhere else we default the optional parameter to false:

https://github.com/puppetlabs/puppet/pull/5883/files#diff-40bfd58bbeef98e0cdbe69d6a84d17e5R120
https://github.com/puppetlabs/puppet/pull/5883/files#diff-ef9407d2777b719bcec044e1e15ef5eaR295
https://github.com/puppetlabs/puppet/pull/5883/files#diff-f0460bb812c9b3d7dcf6d83c6eb70310R57
https://github.com/puppetlabs/puppet/pull/5883/files#diff-d2a368760925bf50f080e11ac668aef2R46

And then we set it to true in the various applications that call into the PMT related code, e.g. https://github.com/puppetlabs/puppet/pull/5883/files#diff-a410247f3ed759df4a60550e388bbd6cR76.